python/hoomd_dlext.cc

```diff
@@ -0,0 +1,80 @@
+// SPDX-License-Identifier: MIT
+// This file is part of `hoomd-dlext`, see LICENSE.md
+
+#include "PyDLExt.h"
+#include "Sampler.h"
+
+namespace py = pybind11;
+using namespace dlext;
+
+void export_SystemView(py::module& m)
+{
+    using SystemViewSPtr = std::shared_ptr<SystemView>;
+    using PyObject = py::object;
+
+    py::class_<SystemView, SystemViewSPtr>(m, "SystemView")
+        .def(py::init<SystemDefinitionSPtr>())
+        .def("is_gpu_enabled", &SystemView::is_gpu_enabled)
+        .def("local_particle_number", &SystemView::local_particle_number)
+        .def("global_particle_number", &SystemView::global_particle_number)
```

Comment:
Right now, we are not exposing `particle_data()` here.
But we use it in PySAGES to get the box information.

Do we want to expose the particle data here, or do we want to pass through the box information?

Reply:
Oh, I removed it accidentally. I want to preserve the option of doing it from here.